src/gluonts/model/trivial/mean.py

```diff
@@ -112,12 +112,11 @@ def train(
         training_data: Dataset,
         validation_dataset: Optional[Dataset] = None,
     ) -> ConstantPredictor:
-        contexts = np.broadcast_to(
```

Comment:
This doesn't look like it belongs to this PR.

Reply:
This is needed in the PR, since `broadcast_to` requires using `len` on the `Dataset` (when it's not strictly needed), but the method may not be available being `Dataset` an `Iterable`.

Reply:
That's mypy who asked me to do this